zebra-state/src/lib.rs

```diff
@@ -87,10 +87,12 @@ pub enum Response {
         /// The hash of the block at the tip of the current chain
         hash: BlockHeaderHash,
     },
-    /// The  response to a `GetKnownBlockHashes` request
-    KnownBlockHashes {
-        /// the hashes of all known blocks that were requested
-        hashes: Vec<BlockHeaderHash>,
+    /// The response to a `Contains` request indicating that the given has is in
+    /// the current best chain
+    Contained {
+        /// The number of blocks above the given block in the current best chain
+        // TODO(jlusby): should we add a type for relative block heights or block distances?
+        depth: BlockHeight,
```

Comment:
```suggestion
        depth: Option<usize>
```
The returned value needs to be an option so that we can signal that a block is not contained in the chain.  Otherwise it's not possible to distinguish a service error from a "not in chain" result.